### DISCUSSION TITLE
Copilot not suggesting anything after login fix

### DISCUSSION THREAD

**Discussion by globalflea** [community/community#40130] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Earlier today, i couldn't login to copilot.  After github fixed the login issue, it automatically logged me in - however, it stop giving any code suggestions.  My IDE is Goland

**Comment by ossanna16** [comment]
Hi there @globalflea and welcome to our community! Thank you for reporting this bug!

**Comment by iamcjwilliams** [comment]
@globalflea Sorry for the trouble here. Are you still experiencing issues? Are you able to provide a screenshot of an error you're receiving?
